### PR TITLE
Log mermaid puppeteer output to console.log

### PR DESCRIPTION
**Pull request by DKurilo**
## :bookmark_tabs: Summary

This PR makes it possible to see log messages in console.
Right now log.info and other messages are invisible as I wrote it here:
https://github.com/mermaid-js/mermaid/issues/1414#issuecomment-1273960484

## :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid-cli/blob/master/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [x] :bookmark: targeted `master` branch

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -185,6 +185,12 @@ async function parseMMD (...args) {
  */
 async function renderMermaid (browser, definition, outputFormat, { viewport, backgroundColor = 'white', mermaidConfig = {}, myCSS, pdfFit } = {}) {
   const page = await browser.newPage()
+  page.on('console', async (msg) => {
+    const msgArgs = msg.args()
+    for (let i = 0; i < msgArgs.length; ++i) {
+      console.log(await msgArgs[i].jsonValue())
+    }
+  })
```
Comment (aloisklink) [inline src/index.js:193]: I did a quick test and this seems to work better, but the Puppeteer docs are a bit vague, so I'm not sure if it is better: https://pptr.dev/api/puppeteer.consolemessage

What do you think?

```suggestion
  page.on('console', (msg) => {
    console.log(msg.text())
  })
```

**Review by aloisklink — APPROVED** [review]
Thanks for adding this feature! It will definitely make debugging a lot easier.

Plus, it means setting `logLevel` in the Mermaid config actually does something now! https://mermaid-js.github.io/mermaid/#/Setup?id=loglevel

See my comment about a potential simplification in the code (I'm not 100% sure if it does the same thing, so let me know if it won't work, or if you have a reason for using `jsonValue()`).

**Comment by aloisklink** [comment]
By the way, I've labelled your PR as `hacktoberfest-accepted` in case you're participating in https://hacktoberfest.com/ for the free T-shirt/tree planted.

**Comment by DKurilo** [comment]
> By the way, I've labelled your PR as `hacktoberfest-accepted` in case you're participating in https://hacktoberfest.com/ for the free T-shirt/tree planted.

Thank you!

File: src/index.js (hunk 1 above)
Comment (DKurilo) [reply src/index.js]: Yeah, it works even better. Thank you.

**Comment by MindaugasLaganeckas** [comment]
@DKurilo thank you for the PR 😄 
Why is docs/animated-flowchart.svg changed? If there is no good reason for the change, please, revert it. 
I have no other comments 😄

**Comment by DKurilo** [comment]
@MindaugasLaganeckas , huh. Thanks. I didn't do it manually. :)
Reverted.

**Comment by aloisklink** [comment]
> Why is docs/animated-flowchart.svg changed?

It's probably because Mermaid v9.1.7 changed the output format. The way I wrote the tests is to keep it updated, if possible, since it's in the README.md file:

https://github.com/mermaid-js/mermaid-cli/blob/22d5e21abf16d80e6b2f51501fd8cb5fc0600da5/src-test/test.js#L204-L207

Maybe we should remove this line from the tests if it's not useful. We can always manually update the file is Percy visual regression tests show it looks different.

**Comment by MindaugasLaganeckas** [comment]
Well I see the problem, but also a value in a test. We may forget to update the doc when/if the test fail.
I would suggest to leave it for now and see how big of a problem it is down the road.
If we get annoyed by changes in docs/animated-flowchart.svg with other PRs, then we will take action.
@aloisklink : what do you think?

**Comment by aloisklink** [comment]
> I would suggest to leave it for now and see how big of a problem it is down the road.
> If we get annoyed by changes in docs/animated-flowchart.svg with other PRs, then we will take action.
> @aloisklink : what do you think?

It sounds good to me!

**Pull request merged**